src/python_testing/TC_ICDM_2_1.py

```diff
@@ -274,8 +274,9 @@ async def test_TC_ICDM_2_1(self):
 
             asserts.assert_less(
                 operatingMode, modes.kUnknownEnumValue, "OperatingMode can only have 0 and 1 as valid values")
+
         self.step(11)
-        if self.pics_guard(self.check_pics("ICDM.S.A0009")):
+        if attributes.MaximumCheckInBackOff.attribute_id in attribute_list:
```

Comment:
Same comment here as on the opstate tests - this gets rid of the skip marker. Though this test is already a mixed bag of whether that happens.

Reply:
Hi Cecille,
I have created a function in MatterBaseTest class named "attributes_guard" as you had mentioned using the check_attributes function in the matter_testing_scripts helper module. 
This should work I hope, please see the update to the issue [#370](https://github.com/project-chip/matter-test-scripts/issues/370) you have filed and let me know if this will resolve the above comment. 
Thank you!

Reply:
is that in this PR or a different PR?

Reply:
Hi Cecille,
Sorry, the above created attributes_guard function is actually implemented currently in PR #[34290](https://github.com/project-chip/connectedhomeip/pull/34290) for the Opstate PICS replacement tests.